layout/_partial/search.ejs

```diff
@@ -4,7 +4,7 @@
             <span class="close-button"><i class="fa fa-times"></i></span>
             <a href="https://algolia.com" target="_blank" rel="noopener" class="searchby-algolia text-color-light link-unstyled">
                 <span class="searchby-algolia-text text-color-light text-small">by</span>
-                <img class="searchby-algolia-logo" src="https://www.algolia.com/static_assets/images/press/downloads/algolia-light.svg">
+                <img class="searchby-algolia-logo" src="assets/images/logo-algolia-nebula-blue-full.svg">
```

Comment:
I think we are not even required to put their logo now so I will check that out and do the necessary.

Reply:
Cool. If that's the case then it can probably just be removed and we won't need to worry.

Reply:
It looks like the logo is still required for OSS: https://www.algolia.com/for-open-source/ (under the FAQ)